desktop/cmp/tab/pane/TabModel.js

```diff
@@ -18,7 +19,9 @@ import {startCase} from 'lodash';
 @HoistModel
 export class TabModel {
     id;
-    title;
+    @bindable title;
+    @observable disabled;
+    @bindable excludeFromSwitcher;
```

Comment:
Does excludeFromSwitcher need to be bindable?  Very curious to see the app that needs this.

Reply:
It doesn't, I made it bindable for symmetry with the other props and because there is no real reason that it _can't_ be bindable. I would have no objections to removing the bindable here if we think it is a simpler/clearer api without it and don't think there would be any real-world use-cases for it.